tests/e2e/test_utils.go

```diff
@@ -28,6 +28,8 @@ import (
 )
 
 const (
+	starterDistType      = "starter"
+	ollamaDistType       = "ollama"
```

Comment:
ollama does not really exist anymore as a distro, I'm not sure it's worth keeping it... Can we remove it in favor of starter?

Reply:
Can do, I was under the impression that the llsd operator should maintain backwards compat with older distro images but I'm happy to remove it if we don't need it.

Reply:
ollama distro hasn't been updated since 0.2.9 😓 so yeah let's remove it, thanks!